packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -1,53 +1,161 @@
 import React from 'react';
+import { isEqual } from 'lodash';
 
 import { Logger } from 'aws-amplify';
-import { changePassword, translate } from '@aws-amplify/ui';
+import {
+  changePassword,
+  FieldValidator,
+  FormEventType,
+  getConfirmPasswordValidator,
+  getDefaultPasswordValidators,
+  runFieldValidators,
+  translate,
+} from '@aws-amplify/ui';
 
 import { useAuth } from '../../../internal';
 import { View, Flex } from '../../../primitives';
+import { FormValues, BlurredFields, ValidationError } from '../types';
 import {
+  DefaultConfirmPassword,
   DefaultCurrentPassword,
   DefaultError,
   DefaultNewPassword,
   DefaultSubmitButton,
 } from './defaultComponents';
 import { ChangePasswordProps } from './types';
-import { FormValues } from '../types';
 
 const logger = new Logger('ChangePassword');
 
+const getIsDisabled = (
```

Comment:
Pulled this outside `ChangePassword` because it's really an util that doesn't need to be redefined everytime.